grep

```diff
@@ -0,0 +1 @@
+palvelutarjotin/__init__.py __version__
```

Comment:
What is the reason for this "grep" file? Is it being used by some other functionality somewhere?

Reply:
it is a mistake, will remove